types/module/server/manager.go

```diff
@@ -52,8 +52,9 @@ func NewManager(baseApp *baseapp.BaseApp, cdc *codec.ProtoCodec) *Manager {
 		router: &router{
 			handlers:         map[string]handler{},
 			providedServices: map[reflect.Type]bool{},
-			antiReentryMap:   map[string]bool{},
+			legacyRouter:     baseApp.Router(),
```

Comment:
I asked this question yesterday during Marie's walkthrough: instead of having the ADR-033 router fall back to the legacy router, can it fallback to baseapp's MsgServiceRouter?

Reply:
I believe that could work too